src/main/java/cz/it4i/parallel/runners/PBSHPCSchedulerBridge.java

```diff
@@ -83,4 +89,9 @@ public void stop(SshCommandClient client, String jobId) {
 		client.executeCommand("qdel " + jobId);
 	}
 
+	private String createOutputDirectoryIfItDoesNotExist() {
```

Comment:
Consider use && instead of ; - if creation fails than it is reasonable not to submit  job.
Consider using mkdir -p instead of mkdir

Reply:
Made the changes.